### DISCUSSION TITLE
Integration with Next.js?

### DISCUSSION THREAD

**Discussion by yanickrochon** [socketio/socket.io#4543] [Q&A] [answered]
I've been searching for this, but can only find outdated articles and blog posts that only make things confusing! How can this integrate with a Next.js app and TypeScript? Considering the popularity of certain platform, such as Next.js, and considering that other people are looking for this feature as well, an example should be provided for at least that framework.

**Comment by darrachequesne** [comment]
That would be great indeed :+1: I'm looking into this.

**Reply by darrachequesne** [reply, marked as answer]
Here we go (better late than never): https://socket.io/how-to/use-with-nextjs
